### trace_handler::main event: call
 0:   @__main__:test_loop => sys_settrace_loop.py:41
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:43
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:44
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:45
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:44
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:45
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:44
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:45
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:44
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:45
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:46
 1:   @__main__:<module> => sys_settrace_loop.py:58
test_for_loop 3
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:49
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:50
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:51
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:53
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:52
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:53
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:52
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:53
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:52
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:53
 1:   @__main__:<module> => sys_settrace_loop.py:58
### trace_handler::main event: line
 0:   @__main__:test_loop => sys_settrace_loop.py:54
 1:   @__main__:<module> => sys_settrace_loop.py:58
test_while_loop 3
### trace_handler::main event: return
 0:   @__main__:test_loop => sys_settrace_loop.py:54
 1:   @__main__:<module> => sys_settrace_loop.py:58
Total traces executed:  23
